contracts/wrapper/MetaERC20Wrapper.sol

```diff
@@ -247,24 +242,25 @@ contract MetaERC20Wrapper is ERC1155Meta, ERC1155MintBurn {
     /* solium-disable-next-line security/no-inline-assembly */
     assembly {
```

Comment:
I wonder if we could simplify this checkSuccess function. What do you think @Agusx1211?

Reply:
It could be written in Solidity, in the same way that `UniswapV2` does it https://github.com/Uniswap/uniswap-v2-core/blob/4dd59067c76dea4a0e8e4bfdda41877a6b16dedc/contracts/UniswapV2Pair.sol#L46. But it wouldn't be more efficient than the current implementation

Reply:
Nice. Kind of neat, but I guess I prefer not touching this since the UniswapV2 has a bit of complexity too anyway. Good to keep in mind for future contracts!